Classes/Settings/SettingsViewController.swift

```diff
@@ -46,6 +46,8 @@ GitHubSessionListener {
     @IBOutlet weak var pushSwitch: UISwitch!
     @IBOutlet weak var pushCell: UITableViewCell!
     @IBOutlet weak var pushSettingsButton: UIButton!
+    @IBOutlet weak var openExternalLinksLabel: SettingsLabel!
```

Comment:
Is this needed? Do you use this variable anywhere?

Reply:
Nope
Maybe it will be needed later for localization. But okay, now I remove it